proprietary/Components/src/denhaag/file.tokens.json

```diff
@@ -0,0 +1,32 @@
+{
+  "denhaag": {
+    "file": {
+      "border": { "value": "1px solid {denhaag.color.grey.2}" },
+      "hover": {
+        "color": { "value": "{denhaag.color.blue.4}" },
+        "background-image": {
```

Comment:
Is this background-image still in use? I don't see it in the css, or did I overlook it?